### ISSUE TITLE
SA1119 Suggests to eliminate meaningful parentheses in pattern

### ISSUE THREAD

**Issue by WilhelmZapiain** [DotNetAnalyzers/StyleCopAnalyzers#2530]
The following code:
```C#
    !(item is null) && SomePredicate(item.Property)
```
triggers SA1119 with the proposed change:
```C#
    !item is null && SomePredicate(item.Property)
```
which is has the wrong semantics.

SA1119 should not trigger in this case.

**Comment by sharwell** [comment]
❓ What version of StyleCop Analyzers are you using? I would expect this to be fixed along with #2372 for release 1.1.0-beta004.

**Comment by WilhelmZapiain** [comment]
Updating to release 1.1.0-beta004 removes StyleCop from the available analyzers. How do I set it up?

**Comment by sharwell** [comment]
What version of Visual Studio are you using? 1.1.x require Visual Studio 2015 Update 2.

**Comment by WilhelmZapiain** [comment]
Community 2017 15.3.3

**Comment by sharwell** [comment]
Thanks. Can you try this?

1. Uninstall StyleCop Analyzers
1. Save everything and close Visual Studio
1. Open Visual Studio again, install StyleCop Analyzers 1.1.0-beta004
1. Save everything and restart Visual Studio again

See if we can't "kick it back into working order"

**Comment by WilhelmZapiain** [comment]
Weird heinsenbug. Had already tried, but know it worked.
All is in working order. Thank you very much.
